### PR TITLE
Fixed Issue 20

### PR DESCRIPTION
**Pull request by krisdestruction**

### CODE REVIEW

**Comment by krisdestruction** [comment]
Fixed Issue #20. Also made longitude wrapping faster.

File: geohash.py [hunk 1]
```diff
@@ -75,12 +82,12 @@ def _encode_i2c(lat,lon,lat_length,lon_length):
...
+	# Check Latitude
+	if latitude > 90.0 or latitude < -90.0:
 		raise Exception("invalid latitude.")
-	while longitude < -180.0:
-		longitude += 360.0
-	while longitude >= 180.0:
-		longitude -= 360.0
+
+	# Wrap Longitude
+	longitude = (longitude + 180.0) % 360.0 - 180.0
```
Comment (hkwi) [inline geohash.py]: This calculation always introduce loss of precision. Original code was expected to unlikely run at most case.

Comment (krisdestruction) [inline geohash.py]: The loss of precision isn't related to this change. This change was just to make the computation a 1-liner for cleaner code.

Comment (hkwi) [inline geohash.py]: See:

```
>>> x=(1.0000001+180.0)%360.0 -180.0
>>> x.hex()
'0x1.000001ad7f280p+0'
>>> 1.0000001.hex()
'0x1.000001ad7f29bp+0'
```

Comment (krisdestruction) [inline geohash.py]: @hkwi How about a quick check if it's within bounds. That way it won't affect precision for the most general case.

Comment (hkwi) [inline geohash.py]: The while loop is already a quick check. What are you suggesting?

Comment (krisdestruction) [inline geohash.py]: I'm going to separate the latitude wrapping as a separate pull request

**Comment by krisdestruction** [comment]
@hkwi Let's separate the wrapping issue into another branch here:
https://github.com/krisdestruction/python-geohash/commit/02d2ebf1f1963d47e34ac4cb542a994c8ce1b8e0

Meanwhile is the rest of the PR acceptable?

**Comment by hkwi** [comment]
No.  See #20
